infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -23,7 +23,7 @@
 @Slf4j
 public class LogReplicationMetadataManager {
 
-    private static final String namespace = CORFU_SYSTEM_NAMESPACE;
+    private static final String NAMESPACE = CORFU_SYSTEM_NAMESPACE;
```

Comment:
mmm yeah I thought I had done it previously too. But I checked and it is correctly rebased, I think the only change is that since it's a static it's all caps.